custom_components/yahoofinance/sensor.py

```diff
@@ -178,6 +182,18 @@ def parse_dividend_date(dividend_date_timestamp) -> str | None:
         dividend_date = datetime.fromtimestamp(dividend_date_timestamp,tz=dt_util.DEFAULT_TIME_ZONE)
         dividend_date_date = dividend_date.date()
         return dividend_date_date.isoformat()
+    
+    @staticmethod
+    def parse_market_time(date_timestamp) -> str | None:
+        """Parse DateTime JSON element."""
+
+        date_timestamp = convert_to_float(date_timestamp)
+        if date_timestamp is None or date_timestamp == 0:
+            return None
+
+        market_date = datetime.fromtimestamp(date_timestamp,tz=dt_util.DEFAULT_TIME_ZONE)
+        market_date_date = market_date  #.datetime()
```

Comment:
We can just use `market_date` here.

Reply:
Also the new function is very much like `parse_dividend_date`. Can we refactor/rename that to handle instant case?